src/lib/overlay-language-service-host.ts

```diff
@@ -81,6 +82,30 @@ export class OverlayLanguageServiceHost implements ts.LanguageServiceHost {
     return contents;
   }
 
+  resolveModuleNames(
+    moduleNames: string[],
+    containingFile: string
+  ): ts.ResolvedModule[] {
+    const resolvedModules: ts.ResolvedModule[] = [];
+    for (const moduleName of moduleNames) {
```

Comment:
nit: I'd use `moduleNames.map()` so it's clear that this is a 1-1 transform of all the elements.